lib/generators/rom/repository_generator.rb

```diff
@@ -0,0 +1,34 @@
+require 'generators/rom'
+
+if defined? ROM::Repository
+  module ROM
+    module Generators
+      class RepositoryGenerator < Base
+        def create_repository_file
+          template(
+            'repository.rb.erb',
+            File.join('app', 'repositories', "#{repository_name}_repository.rb")
+          )
+        end
+
+        private
+
+        def relation
+          class_name.pluralize.underscore
+        end
+
+        def model_name
+          class_name.singularize.capitalize
```

Comment:
This should really be camel-case -- consider what happens when we generate a `user_profile` repository.